yarn.lock

```diff
@@ -23435,6 +23944,14 @@ store2@^2.12.0:
   resolved "https://registry.yarnpkg.com/store2/-/store2-2.12.0.tgz#e1f1b7e1a59b6083b2596a8d067f6ee88fd4d3cf"
   integrity sha512-7t+/wpKLanLzSnQPX8WAcuLCCeuSHoWdQuh9SB3xD0kNOM38DNf+0Oa+wmvxmYueRzkmh6IcdKFtvTa+ecgPDw==
 
+storybook-addon-export-to-codesandbox@^0.1.8:
+  version "0.1.8"
+  resolved "https://registry.yarnpkg.com/storybook-addon-export-to-codesandbox/-/storybook-addon-export-to-codesandbox-0.1.8.tgz#6eadd241d29aaaf52907676dfbeb6d7f0db95117"
+  integrity sha512-JL/rTzVCtrB43xNJsHiZHed4Lh4oNu80CBvOITT1JGLJBTSOU1qx9M2PjwvRtxa3YejpTofkzop6h0R9UmTGqQ==
+  dependencies:
+    codesandbox "^2.2.3"
```

Comment:
Can you please use `codesandbox-import-utils` instead?

[`src/withCodeSandboxButton.ts`](https://github.com/PeterDraex/storybook-addon-export-to-codesandbox/blob/main/src/withCodeSandboxButton.ts)

```diff
-import { getParameters } from 'codesandbox/lib/api/define';
+import { getParameters } from 'codesandbox-import-utils/lib/api/define';
```

Having the whole `codesandbox` app in our dependencies is too much 😅

Reply:
sorry about that 😅